db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -2035,6 +2035,25 @@ public class SQLParser
 						    getContextManager());
     }
 
+    /**
+     * Get a left() node from
```

Comment:
s/Get/Gets

Reply:
I fixed the typos, thanks! Also, I configured the IDE to remove trailing spaces on save, thus many chances in the file.